openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -1015,7 +1015,6 @@ OS:SizingPeriod:DesignDay,
        \note Humidity Indicating Type
        \note required-field if Humidity Indicating Day Schedule is not used
        \type real
-       \units BasedOnField A3
```

Comment:
See #1897. Turns out that was only a bug when in IP units, which brought me to these changes.